lua/telescope/builtin/git.lua

```diff
@@ -47,6 +47,7 @@ git.commits = function(opts)
 end
 
 git.bcommits = function(opts)
+  opts.relative_file_path = vim.fn.expand('%')
```

Comment:
lets do `opts.current_file = opts.current_file of vim.fn.expand('%')` that way someone could just call `require'telescope.builtin'.bcommits{ current_file = 'path/to/file.lua }'` no matter which file is currently open. That could be cool

Reply:
Great idea!